packages/mint-components/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.2] - 2022-03-04
```

Comment:
Date is not correct if you are trying to deploy this today

Reply:
👍 fixed